instantsearch/src/main/java/com/algolia/instantsearch/helpers/Searcher.java

```diff
@@ -929,6 +992,17 @@ public void requestCompleted(JSONObject content, AlgoliaException error) {
         });
     }
 
+    @Override public String toString() {
+        String key = null;
```

Comment:
For cases like this we could use BiMap in the future to avoid writing redundant code. https://google.github.io/guava/releases/19.0/api/docs/com/google/common/collect/BiMap.html

Reply:
👍, adding a todo for later refactoring then!